server/text_generation_server/layers/fp8.py

```diff
@@ -12,17 +12,26 @@
     Weights,
 )
 from text_generation_server.utils.log import log_master, log_once
+import importlib.util
+
 
 FBGEMM_MM_AVAILABLE = False
 FBGEMM_DYN_AVAILABLE = False
-try:
-    import fbgemm_gpu.experimental.gen_ai
 
+
+def is_fbgemm_gpu_available():
+    try:
+        return importlib.util.find_spec("fbgemm_gpu.experimental.gen_ai") is not None
```

Comment:
What's the rationale behind this change ?

I find it must less clear in intent. If ruff complains there must be a good reason behind it.